FFXIVClientStructs/FFXIV/Client/Graphics/Render/Material.cs

```diff
@@ -1,9 +1,44 @@
+using FFXIVClientStructs.FFXIV.Client.Graphics.Kernel;
 using FFXIVClientStructs.FFXIV.Client.System.Resource.Handle;
 
 namespace FFXIVClientStructs.FFXIV.Client.Graphics.Render; 
 
 [StructLayout(LayoutKind.Explicit, Size = 0x20)]
```

Comment:
Find the new size for this, as it overruns the size constraints with the field offsets

Reply:
Good catch, it is indeed `0x40` + a variable part, fixed it + added a comment.